cognite/client/__init__.py

```diff
@@ -1,3 +1,3 @@
 from cognite.client._cognite_client import CogniteClient
 
-__version__ = "1.8.0"
```

Comment:
think we can have the same version for unique values and STS?

Reply:
Yes, I undid the version change to bulk it